03_launch_mgmt_cluster.sh

```diff
@@ -219,6 +224,37 @@ EOF
   popd
 }
 
+#
+# Launch and configure fakeIPA
+#
+function launch_fakeIPA() {
+  # Create a folder to host fakeIPA config and certs
+  mkdir -p /opt/metal3-dev-env/fake-ipa
+  if [[ "${EPHEMERAL_CLUSTER}" == "kind" && "${IRONIC_TLS_SETUP}" == "true" ]]; then
+    cp "${IRONIC_CACERT_FILE}" "/opt/metal3-dev-env/fake-ipa/ironic-ca.crt"
+  elif [[ "${IRONIC_TLS_SETUP}" == "true" ]]; then
+    # wait for ironic to be running to ensure ironic-cert is created 
+    kubectl -n baremetal-operator-system wait --for=condition=available deployment/baremetal-operator-ironic --timeout=900s
+    # Extract ironic-cert to be used inside fakeIPA for TLS 
+    kubectl -n legacy get secret -n baremetal-operator-system ironic-cert -o json -o=jsonpath="{.data.ca\.crt}" | base64 -d > /opt/metal3-dev-env/fake-ipa/ironic-ca.crt
```

Comment:
Typo?
```suggestion
    kubectl get secret -n baremetal-operator-system ironic-cert -o json -o=jsonpath="{.data.ca\.crt}" | base64 -d > /opt/metal3-dev-env/fake-ipa/ironic-ca.crt
```

Reply:
nice catch cool that it was still working 😆